libavformat/mxfdec.c

```diff
@@ -4428,8 +4431,8 @@ static const AVOption options[] = {
     { "eia608_extract", "extract eia 608 captions from s436m track",
       offsetof(MXFContext, eia608_extract), AV_OPT_TYPE_BOOL, {.i64 = 0}, 0, 1,
       AV_OPT_FLAG_DECODING_PARAM },
-    { "dovi_metadata_extract", "extract Dolby Vision metadata",
-      offsetof(MXFContext, dovi_metadata_extract), AV_OPT_TYPE_BOOL, {.i64 = 0}, 0, 1,
+    { "dovi_metadata_extract", "extract Dolby Vision (ie. Prototype HDR) metadata",
```

Comment:
My thoughts here are that within the scope of the MXF demuxer, the DoVi metadata is encapsulated by the PHDR track components.  To an outside consumer, they should only be concerned with enabling the extraction of DoVi metadata, since that will be present within the AVStream after it's pulled out of the MXF.